lib/smart_proxy_dns_infoblox/plugin_configuration.rb

```diff
@@ -0,0 +1,19 @@
+module Proxy::Dns::Infoblox
+  class PluginConfiguration
+    def load_classes
+      require 'infoblox'
+      require 'dns_common/dns_common'
+      require 'smart_proxy_dns_infoblox/dns_infoblox_main'
+    end
+
+    def load_dependency_injection_wirings(container_instance, settings)
+      container_instance.dependency :connection,
+                                    lambda {::Infoblox::Connection.new(:username => settings[:infoblox_user] ,:password => settings[:infoblox_pw], :host => settings[:infoblox_host])}
```

Comment:
So we should rename the settings here as well, or? What about the server option unification?